### ISSUE TITLE
Adding analytics for the application

### ISSUE THREAD

**Issue by sonalranjit** [bikespace/bikespace#73]
- [x] Landing page

  - completed in #97 

- [x] Dashboard

  - completed in #97 

- [x] Frontend

**Comment by tallcoleman** [comment]
Metrics that might be helpful for the app:
- completion rate for sessions that get past the first page
- drop-off rate by form page (to determine if there are any pages that users are struggling more with)

**Comment by sonalranjit** [comment]
Some open source alternatives to Google Analytics: 
- https://umami.is/
- https://plausible.io/
- https://matomo.org/

Leaning towards umami, mostly because we don't need anything too detailed, and they have a hobby tier for 3 websites.

**Comment by tallcoleman** [comment]
Tried setting up Umami on my personal blog and setting up an event and it looks fantastic. Couldn't be easier to use!

**Comment by tallcoleman** [comment]
If we add Umami, we should do it like this so that local development doesn't get included in the analytics:

```js
// HOST_URL could be bikespace.ca, dashboard.bikespace.ca, or app.bikespace.ca
// WEBSITE_ID from the Umami settings for the website
// Note that src may be different if you choose US hosting instead of Europe
if (document.location.host === "HOST_URL") { 
  let s = document.createElement("script");
  s.src = "https://eu.umami.is/script.js";
  s.dataset["websiteId"] = "WEBSITE_ID"
  document.head.appendChild(s);
}
```

**Comment by hulloanson** [comment]
Closed by #97 ?
